lightstep/propagation.py

```diff
@@ -11,10 +11,15 @@ class LightStepFormat(object):
     These static constants are intended for use in the Tracer.inject() and
     Tracer.extract() methods. E.g.,
 
-        tracer.inject(span.context, LightStepFormat.LIGHTSTEP_BINARY, lightstep_carrier)
+        tracer.inject(
+            span.context, LightStepFormat.LIGHTSTEP_BINARY, lightstep_carrier
+        )
 
     """
 
     # The LIGHTSTEP_BINARY format represents SpanContexts in byte array format.
     # https://github.com/lightstep/lightstep-tracer-common/lightstep_carrier.proto
     LIGHTSTEP_BINARY = 'lightstep_binary'
+    # The LIGHTSTEP_B3 format represents SpanContexts in B3 format.
+    # https://github.com/openzipkin/b3-propagation
+    LIGHTSTEP_B3 = 'lightstep_b3'
```

Comment:
This can be called `B3` (`LIGHSTEP_BINARY` has this name because it's a binary specific format that only us use ;) )

Reply:
I removed it completely from this class since it is specified to be only for LightStep-specific carrier formats. The module says it is also for this kind of carrier formats. Should the B3 specification identifier declaration (`B3 = "b3"`) be on a new file?